src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    public static final int OPERATION_TYPE = 0;
+    public static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        int supplySum = 0;
+        int buySum = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                String[] words = value.split(",");
+                if (words[OPERATION_TYPE].equals("supply")) {
+                    supplySum += Integer.parseInt(words[AMOUNT]);
+                } else if (words[OPERATION_TYPE].equals("buy")) {
+                    buySum += Integer.parseInt(words[AMOUNT]);
+                }
+                value = bufferedReader.readLine();
+            }
```

Comment:
Instead of using `substring()`, it's recommended to use `split()` to make the code cleaner. However, you are already using `split()` which is good. Just make sure to avoid `substring()` in future implementations if applicable.